### ISSUE TITLE
No Name is provided for "All color Delete button" in Categories and Tags screen: A11y_.NET 9 template_Manage meta_Name Role Value.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26918]
### GitHub Tags:
#A11yTCS; #A11ySev1; #Android; #WCAG4.1.2; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #A11yMAS; #Name Role Value; #AILimited; #Device; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback version:15.1

### Repro Steps:

1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till Hamburger menu and double tap on it, Swipe till manage meta and double tap on it, Categories and tags screen will open,
4. We can add additional categories and tags or delete it.
5. We can give the name and color to the categories and tags.
6. Swipe till Delete button and Observe that No Name is provided for "All color Delete button" in Categories and Tags screen.

### Actual Result:
No Name is provided for "All color Delete button" in Categories and Tags screen.

### Expected Result:
Name should be provided for "All color Delete button" in Categories and Tags screen.

### User Impact:
Users who relay on screen reader wont get proper information of that delete button.

### Attachment:
https://github.com/user-attachments/assets/6ed6510e-8137-41d7-8f9d-62b5fe479b05

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by keerthiduvvuri** [comment]
Verified in latest Environment, Issue is Fixed. Hence closing the bug.
Added attachment for reference.
#Regressed:03-06-25; #Closed; #A11yFixed;

https://github.com/user-attachments/assets/d9a38e14-6c06-44a0-8651-ecf161fe75d6
